### PR TITLE
Antd trends updated layout

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

- finishes #532 

## Checklist
~~- [ ] All querysets/queries filter by Team~~
- [ ] Code reviewed
- [ ] QA'd

### CODE REVIEW

File: frontend/src/lib/hooks/useWindowSize.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react'
+
+export function useWindowSize() {
```
Comment (EDsCODE) [inline frontend/src/lib/hooks/useWindowSize.js:3]: REF: https://usehooks.com/useWindowSize/

Could use react-responsive but refrained from adding more packages atm

**Review by timgl — COMMENTED** [review]
Very nice!

The Card bar isn't aligned correctly. 
image


The whole layout looks a little... off on a larger screen like mine. I wonder if the solution is maybe to remove the `min-height: 85vh`

image

To this:
image

**Comment by EDsCODE** [comment]
the minheight is meant to make the cards the same height for consistency purposes. We could let the items within the cards determine the height too but I just thought the consistent heights looked a bit more put-together. I changed the condition to display specifically when the window is longer horizontally than vertically so on long vertical windows it should switch back to our original layout!

and adjusted the padding for the card bar (I was fooling with it because I wanted the bottom of that grey container to align with the separator under the tabs on the settings card)

**Comment by timgl** [comment]
I had a quick play around with this. I think the actual heuristic that makes more sense is that if the height of the screen is > 1200, it should _not_ have the min-height. That way if the screen is wide as well as tall it will still be collapsed.

**Comment by EDsCODE** [comment]
Removed minheight! Card heights will be sized by content inside so there shouldn't be elongated columns

**Review by timgl — APPROVED** [review]

**Pull request merged**